diga-epa-client/src/main/java/com/oviva/epa/client/internal/svc/PhrManagementServiceClient.java

```diff
@@ -17,14 +19,31 @@ public PhrManagementServiceClient(
     this.konnektorContext = konnektorContext;
   }
 
+  public GetAuthorizationStateResponse getAuthorizationState(String knvr, String homeCommunityId) {
+    var context = getContext();
+
+    var req =
+        new GetAuthorizationState()
+            .withContext(context)
+            .withUserAgent("PS_123/V1.1.0/gematik") // TODO where does that come from?
```

Comment:
it doesn't work without this user agent? if so how did you come up with it? an exact google search returns 0 results :)

Reply:
Found the answer, see comments :)